integration_testing/features/app-context/app-context-disable-device-network-access.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Disabled network access to Kolibri app server
+	If the network access to Kolibri server app is disabled, users who navigate to its URL need to see the notification message
+
+Background:
+	Given that the device setting *Allow others in the network to access Kolibri on this device using a browser* is unchecked
+		And I use an external device to connect to the Kolibri app server and access Kolibri
```

Comment:
You don't need to be in an external device. An incongnito window in the browser or any other browser where the cookie for the App user is not set should trigger the error. Maybe asking to use an incognito window makes this test easier to be implemented.

Reply:
Good point, will add a comment bellow!